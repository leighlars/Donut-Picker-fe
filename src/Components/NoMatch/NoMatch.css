.NoMatch {
  background-image: url('../../assets/closed2.jpeg');
  background-size: 100%;
  filter: blur(6px);
}