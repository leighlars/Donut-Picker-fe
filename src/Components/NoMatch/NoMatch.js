import React from 'react';
import { Link } from 'react-router-dom';
import image from '../../assets/sprinkleboy.jpeg';

export default function NoMatch() {
  return (
    <section className="NoMatch">
      <section className="no-match-link">
        <h3>Sorry! That page doesn't exist.</h3>
        <Link className="back-btn" to='/'>
          <button>Go Back</button>
        </Link>
      </section>
      <img src={image} alt="boys face covered in sprinkles" />
    </section>
  )
}
