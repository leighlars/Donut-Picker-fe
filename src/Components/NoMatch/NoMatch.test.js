import React from 'react'
import {shallow} from 'enzyme'
import NoMatch from './NoMatch'


describe('NoMatch', () => {
    let wrapper;
    wrapper = shallow( <NoMatch/> )
    it('should matchsnapshot', () => {
        expect(wrapper).toMatchSnapshot()
    });
    
});