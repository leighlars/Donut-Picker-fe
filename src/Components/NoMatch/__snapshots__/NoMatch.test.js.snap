// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should matchsnapshot 1`] = `ShallowWrapper {}`;
